CHANGELOG.md

```diff
@@ -14,7 +14,18 @@
         self.startNavigation(with: route, animated: false)
     #endif
     ```
-* Custom location snapping in the `RouteController` via the delegate
```

Comment:
This was https://github.com/maplibre/maplibre-navigation-ios/pull/85 - so it should not be removed :)

Reply:
Its not removed, its on line 27, including credit to the author (I replaced it by the autogenerated notes by github) ☺️

Reply:
Sorry, I missed that 👍